src/utils/querybuilder.util.js

```diff
@@ -561,6 +567,9 @@ const numberQueryBuilder = function ({ target, field }) {
  */
 const quantityQueryBuilder = function ({ target, field }) {
     const qB = {};
+    if (!target || !(typeof target === 'string') || target.length === 0) {
```

Comment:
Please add testcases for quantity type too for 'missing' modifier

Reply:
Not doing quantity type in this JIRA